### PR TITLE
updates a broken link

### PR DESCRIPTION
**Pull request by OnkarRuikar**

### CODE REVIEW

File: files/en-us/web/security/securing_your_site/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ There are a number of things you can do to help secure your site. This article o
...
-  - : An added layer of security that helps to detect and mitigate certain types of attacks, including {{Glossary("Cross-site_scripting", "Cross Site Scripting (XSS)")}} and data injection attacks. These attacks are used for everything from data theft to site defacement or distribution of malware. Code is executed by the victims and lets the attackers bypass access controls and impersonate users. According to the Open Web Application Security Project, XSS was the [seventh most common Web app vulnerability](https://owasp.org/images/7/72/OWASP_Top_10-2017_%28en%29.pdf.pdf) in 2017.
+  - : An added layer of security that helps to detect and mitigate certain types of attacks, including {{Glossary("Cross-site_scripting", "Cross Site Scripting (XSS)")}} and data injection attacks. These attacks are used for everything from data theft to site defacement or distribution of malware. Code is executed by the victims and lets the attackers bypass access controls and impersonate users. According to the Open Web Application Security Project, XSS was the [third most common Web app vulnerability](https://owasp.org/www-project-top-ten/) in 2021.
```
Comment (wbamberg) [inline files/en-us/web/security/securing_your_site/index.md:32]: Perhaps rather than update this every year or so we should just delete this sentence?

Comment (teoli2003) [reply files/en-us/web/security/securing_your_site/index.md:32]: Maybe we want to link to https://owasp.org/www-community/attacks/xss/ instead (still, need rephrasing)

**Comment by OnkarRuikar** [comment]
I think it is good to have the statement to emphasize the seriousness of the attack. How about we make it nonspecific by removing the position and year. So it'll remain true for longer.
>  According to the Open Web Application Security Project, XSS is the [most common Web app vulnerability](https://owasp.org/www-project-top-ten/).

***
@wbamberg The last link was to a pdf file so it broke easily, but here we are changing it to html page derived from https://github.com/OWASP/www-project-top-ten/blob/master/index.md. So it is not likely to break.

@teoli2003 we are already providing a link to our XSS glossary page in the same paragraph. How about we provide the owsp xss link in see also section?

File: files/en-us/web/security/securing_your_site/index.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ There are a number of things you can do to help secure your site. This article o
...
 - [Content Security Policy](/en-US/docs/Web/HTTP/CSP)
-  - : An added layer of security that helps to detect and mitigate certain types of attacks, including {{Glossary("Cross-site_scripting", "Cross Site Scripting (XSS)")}} and data injection attacks. These attacks are used for everything from data theft to site defacement or distribution of malware. Code is executed by the victims and lets the attackers bypass access controls and impersonate users. According to the Open Web Application Security Project, XSS was the [third most common Web app vulnerability](https://owasp.org/www-project-top-ten/) in 2021.
+  - : An added layer of security that helps to detect and mitigate certain types of attacks, including {{Glossary("Cross-site_scripting", "Cross Site Scripting (XSS)")}} and data injection attacks. These attacks are used for everything from data theft to site defacement or distribution of malware. Code is executed by the victims and lets the attackers bypass access controls and impersonate users. According to the Open Web Application Security Project, XSS is the [most common Web app vulnerability](https://owasp.org/www-project-top-ten/).
```
Comment (teoli2003) [inline files/en-us/web/security/securing_your_site/index.md:32]: Let's be more future-proof.
```suggestion
  - : An added layer of security that helps to detect and mitigate certain types of attacks, including {{Glossary("Cross-site_scripting", "Cross Site Scripting (XSS)")}} and data injection attacks. These attacks are used for everything from data theft to site defacement or distribution of malware. Code is executed by the victims and lets the attackers bypass access controls and impersonate users. According to the Open Web Application Security Project, XSS is one the [most common Web app vulnerability](https://owasp.org/www-project-top-ten/).
```

Comment (OnkarRuikar) [reply files/en-us/web/security/securing_your_site/index.md:32]:
```suggestion
  - : An added layer of security that helps to detect and mitigate certain types of attacks, including {{Glossary("Cross-site_scripting", "Cross Site Scripting (XSS)")}} and data injection attacks. These attacks are used for everything from data theft to site defacement or distribution of malware. Code is executed by the victims and lets the attackers bypass access controls and impersonate users. According to the Open Web Application Security Project, XSS is one of the [most common Web app vulnerability](https://owasp.org/www-project-top-ten/).
```

**Review by teoli2003 — APPROVED** [review]

**Comment by wbamberg** [comment]
> @wbamberg The last link was to a pdf file so it broke easily, but here we are changing it to html page derived from https://github.com/OWASP/www-project-top-ten/blob/master/index.md. So it is not likely to break.
> 

It's not about the link breaking. Presumably https://owasp.org/www-project-top-ten/ won't talk about the position of XSS in a 2021 ranking forever - it will get updated with newer rankings in newer years, and our description of it will rot.

**Comment by teoli2003** [comment]
I call this writing _time-agnostic documentation_, which is a kind of neutrality (to use a Wikipedia term). 😄 I think we can reasonably assume that XSS will stay in the top-10, though.

**Comment by OnkarRuikar** [comment]
So are we keeping the sentence or not?

**Comment by wbamberg** [comment]
> So are we keeping the sentence or not?

I'm on vacation and don't feel that strongly, but would vote to delete. What's it adding? XSS is a serious vulnerability - of course it is, or we wouldn't list it here. The OWASP page lists access control at #1, should we include the same text in the entry for implementing access control? That would be silly, right? But why include it in one and not the other?

I think it's fine, and enough, to include that link in see also.

Also the glossary entry for XSS has the same thing in it: https://developer.mozilla.org/en-US/docs/Glossary/Cross-site_scripting.

**Comment by teoli2003** [comment]
> The OWASP page lists access control at https://github.com/mdn/content/pull/1, should we include the same text in the entry for implementing access control?
No strong feeling here. Let's not add it as this wouldn't scale.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**